### PR TITLE
Bump build-push-action version

### PR DESCRIPTION
**Pull request by eriksjolund**
* bump version to docker/build-push-action@v4

* fix typo

### CODE REVIEW

**Review by talavis — CHANGES_REQUESTED** [review]
The multi arch build is in docker-multiarch.yml

**Comment by eriksjolund** [comment]
Thanks for the review. I reworked the PR to only bump the build-push-action version and to fix a typo.

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Templates for common Docker builds:
 Templates for common workflows:
 
 * Perform a codeql analysis on a code base (`codeql-analysis.yml`)
-* Build and publish Docker images (`docker.yml`)
+* Build and publish Docker images (`docker.yml`). (Optional: Adjust the `platforms` attribute to build images for additional architectures like linux/arm64`)
```
Comment (talavis) [inline README.md:22]: I think you forgot to remove this one.

Comment (eriksjolund) [reply README.md]: You're right. Now fixed.

**Review by talavis — APPROVED** [review]

**Pull request merged**